editor/effects.js

```diff
@@ -276,4 +280,18 @@ export default {
 
 		return effects;
 	},
+	METABOX_LOADED( action, store ) {
+		const { getState } = store;
+		const metaboxes = getMetaBoxes( getState() );
+		const unloadedMetaboxes = filter(
+			map( metaboxes, ( value, key ) => ( {
+				...value,
+				key,
+			} ) ),
+			( metabox ) => metabox.isActive && ! metabox.isLoaded
+		);
+		if ( unloadedMetaboxes.length === 1 && unloadedMetaboxes[ 0 ].key === action.location ) {
```

Comment:
Makes me wonder whether it was a wise decision in `refx` to handle effects prior to the original action behavior. As a side-effect it seems intuitive that it occurs _after_ the original action. Or at least wondering if it should have been offered as a choice (before/after, or explicit next handling).

Reply:
I thought the same and hesitated about adding another middleware or something to do so. But yeah, I think offering the choice might be a good thing if the API stays simple enough.